src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```

Comment:
###AWArSQXN-3vUqknkNC1n###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_24@9
A "NullPointerException" could be thrown; "message" is nullable here.

Reply:
===RESOLVED===
This issue has been resolved by a user in SonarQube